dracut/30ignition/ignition-fetch-offline.service

```diff
@@ -0,0 +1,28 @@
+[Unit]
+Description=Ignition (fetch-offline)
+Documentation=https://github.com/coreos/ignition
+ConditionPathExists=/etc/initrd-release
+DefaultDependencies=false
+Before=ignition-complete.target
+After=basic.target
+
+# Stage order: setup -> fetch-offline [-> fetch] -> disks -> mount -> files.
+# We run after the setup stage has run because it may copy in new/different
+# ignition configs for us to consume.
+After=ignition-setup-base.service
+After=ignition-setup-user.service
+Before=ignition-fetch.service
+
+OnFailure=emergency.target
+OnFailureJobMode=isolate
+
+# See hack below, as well as coreos-copy-firstboot-network.service.
+After=dracut-cmdline.service
+Before=dracut-initqueue.service
```

Comment:
seems like we're going to race with `coreos-copy-firstboot-network.service`. I think we need ot enforce some ordering either here or there.

Reply:
Good catch, made that part of https://github.com/coreos/fedora-coreos-config/pull/426.